internal/cloudapi/openapi.yml

```diff
@@ -110,18 +110,45 @@ components:
       required:
         - status
         - type
```

Comment:
`type` from `UploadStatus` should be probably removed in favor of the `TargetResult`'s `name` property which technically holds the same information and also identifies the type of the upload target.

Reply:
So in image-builder and the cloudapi we use `type` to specify the target in `upload_request`. It's indeed a bit confusing and was just decided earlier on. If we switch to using `name` here we need to change it in all 3 projects for `upload_requests`.